app/views/foods/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @food %>
+
+<div>
+
+    <p><b>Name </b><%= @food.name %></p>
+    <p><b>Measurement Unit: </b></b><%= @food.measurement_unit %></p>
+    <p><b>Price: </b><%= @food.price %></p>
+
+  <%= link_to "Back to foods", foods_path %>
+  <%= button_to "Destroy this food", @food, method: :delete %>
```

Comment:
- [ ] I created a food from account of user1, and logged in with user2 account, I should not be able to delete food, but I can delete food even though it was created by another user. 
**Please make sure that  a food item can be deleted from the lists - only by owners**

Reply:
Sorted in the side [food add PR](https://github.com/kwambiee/rails-recipe-app/pull/33)